docs/editors-guide/rare-disease-subset.md

```diff
@@ -0,0 +1,51 @@
+# Rare Disease Subset in Mondo
+
+A rare disease is a disease that affects fewer than 200,000 people in the United States. However, this definition is not universal, and it varies between rare disease sources, geographical locations, etc. Consequently, rare disease criteria also vary between sources, and one might have difficulties reconciling sources and creating a list for rare diseases. Mondo recognizes these impediments and differences of view. This document explains how Mondo handles rare diseases, how one could get a rare disease list, and the efforts made to bring the community together.
+
+#### What are rare disease in Mondo?
+We rely on rare disease authorities (see table below) to determine all diseases recognized as “rare”, according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annoted with `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority).
+
+Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
+
+![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
+
+Some Mondo terms were annotated with a `‘bearer of’ some rare` axiom. These axioms were added automatically to terms coming from Orphanet (before the creation of 'orphanet_rare' in_subset annotation), and manually by a curator based on the disease definition.
```

Comment:
You only list Orphanet here as the source for "bearer of some rare". Why then not keep this in sync with Orphanet? The only thing that would matter if Mondo curators would add this restriction on a class that was not defined as rare according to anyone else. INn that case, we need to say what Mondo considers rare at all.

Here is an idea: we query for all classes that are rare according to mondo but no one else (https://api.triplydb.com/s/urdlrUxt_), and develop a short narrative for why these should be considered rare despite no one else saying so? I don't think this is worth doing until _after_ there GARD and NORD PRs are merged though.

Reply:
I don't know why we are not keeping it in sync with Orphanet... From my point of view, having "orphanet-rare" subset annotation and "bearer of rare" for orphanet is redundant (or would be redundant if we were keeping it in sync)
IMO, these axioms should be added to "every" rare diseases, or to only diseases that a curator identified as rare (and is not in a rare subset). 

I created an [issue](https://github.com/monarch-initiative/mondo/issues/6448) to discuss this.